zone/trading.cpp

```
@@ -879,18 +961,71 @@ void Client::TraderStartTrader(const EQApplicationPacket *app)
 		}
 
 		trader_items.emplace_back(trader_item);
+		listed_slots.insert(item.slot_id);
+		return true;
+	};
+
+	auto find_satchel_item_by_unique_id = [&satchel_items, &unique_ids_match](const std::string &item_unique_id) -> TraderSatchelItem* {
+		for (auto &item: satchel_items) {
+			if (item.inst && unique_ids_match(item.inst->GetUniqueID(), item_unique_id)) {
+				return &item;
```

Cppcheck (2.21.0)

[style] The function 'TGB' is never used.

(unusedFunction)